src/plugins/sync-font.js

```diff
@@ -0,0 +1,16 @@
+export function clientSyncFont(client) {
+	return client.request('font').then((font) => {
+		document.body.style.fontFamily = font.family;
+		document.body.style.fontSize= font.size;
+	});
+}
+
+export function hostSyncFont(host) {
+	host.onRequest('font', () => {
+		const computedStyle = window.getComputedStyle(document.body);
```

Comment:
Correct, each app would call this once. We could definitely cache it was determined to be a performance issue.